src/EPPlusTest/AmanaIssues.cs

```diff
@@ -480,6 +480,20 @@ public void ExcelPackage_Calculate()
             Assert.AreEqual("123 456 ABC", sheet.Cells["B9"].Value);
         }
 
+        [TestMethod]
+        public void Test_FormulaSumPrecision()
```

Comment:
The test looks simple🙂 — it doesn't have any specific Assertion. It looks like the test's assertion is "Should not throw exception".

@AmanaFranc, did you copy this test from Old EPPlus "As Is" or you make this from scratch?